src/main/scala/mesosphere/marathon/state/MarathonStore.scala

```diff
@@ -118,7 +116,7 @@ class MarathonStore[S <: MarathonState[_, S]](
     // TODO use implicit conversion after it has been merged
```

Comment:
well, maybe you should use the implicit conversion, now ;)

Then you also get your `blocking` call for free ;)